UKHomeOffice/nmsw-ui
# NMSW-436 [pt2] Refactor if statements into a switch and error handler

# Ticket

NMSW-436

----

## AC

- refactor the file upload error handling into a switch statement and handleError function

----

## To test

Unit tests should pass

- run app
- create new voyage
- click submit
- > should error with missing file
- upload image file
- click submit
- > should error with invalid file type
- upload large file
- click submit
- > should error with file too large
- delete auth token
- upload fal file
- click submit
- > should redirect to sign in
- sign in
- upload bad fal file
- click submit
- > should error
- upload good fal file
- click submit
- > should get confirmation page


----

## Notes
